mozilla-release/browser/components/preferences/in-content/preferences.js

```diff
@@ -109,6 +109,15 @@ function init_all() {
   } catch (e) {
     Cu.reportError(e);
   }
+  if (typeof gExperimentsPane !== 'undefined') {
```

Comment:
Should this be wrapped into a comment indicating that this is custom Cliqz code?

Reply:
Follows connect code.